### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Hades32**
* cleans the code A LOT
  * the SDK had direct calls to the plugin and thus made it not reusable anymore
* Java 17 stuff
* uses v2 APIs
* nacks failed commands

Build for testing
[TangiaSpigot-1.0-cleanup.zip](https://github.com/UltimateTournament/MCSpigotPlugin/files/9756075/TangiaSpigot-1.0-cleanup.zip)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,11 +6,13 @@ services:
     ports:
       - 25565:25565
     environment:
+      TANGIA_ENV: "STAGING"
```
Comment (danthegoodman1) [inline docker-compose.yml]: might not be a good idea to put into main if we are asking them to copy this

**Review by danthegoodman1 — CHANGES_REQUESTED** [review]
one small thing, otherwise lgtm

File: docker-compose.yml (hunk 1 above)
Comment (Hades32) [reply docker-compose.yml]: good point

**Comment by Hades32** [comment]
Made a release https://github.com/UltimateTournament/MCSpigotPlugin/releases

**Pull request merged**